cdap-client-tests/src/test/java/co/cask/cdap/client/MetaClientTestRun.java

```diff
@@ -18,21 +18,28 @@
 
 import co.cask.cdap.client.common.ClientTestBase;
 import co.cask.cdap.client.exception.UnAuthorizedAccessTokenException;
+import co.cask.cdap.proto.Version;
 import co.cask.cdap.test.XSlowTests;
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import org.junit.Assert;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
 import java.io.IOException;
 
 /**
- * Test for {@link PingClient}.
+ * Test for {@link MetaClient}.
  */
 @Category(XSlowTests.class)
```

Comment:
This requires standalone CDAP, which takes a while to start up, so this is marked as XSlow.